sentry/sentry.py

```diff
@@ -46,22 +57,59 @@ def cog_unload(self):
         self.bot.remove_before_invoke_hook(self.before_invoke)
         return super().cog_unload()
 
+    @commands.command()
+    @checks.mod()
+    async def sentry_set_env(self, context: commands.context.Context, new_value: str):
+        """Set sentry environment"""
+        if not context.guild:
+            await context.send((
+                "Environment can only be changed from a discord server as "
+                "it's a per-server setting."
+            ))
+            return
+        await self.config.guild(context.guild).environment.set(new_value)
+        await context.send(f"Value of environment has been changed to '{new_value}'!")
+
+    @commands.command()
+    @checks.mod()
+    async def sentry_get_env(self, context: commands.context.Context):
+        """Get sentry environment"""
+        if not context.guild:
+            await context.send((
+                "Environment can only be changed from a discord server as "
+                "it's a per-server setting."
```

Comment:
```suggestion
                "That command is not available in DMs. Sentry environment "
                "is a per-guild setting."
```
This message isn't relevant since nothing is being changed.
We can fix this, match red's functionality when using per-guild commands (`That command is not available in DMs.`), and add some extra information.